docs/guides/functions.ipynb

```diff
@@ -293,7 +302,12 @@
     "\n",
     "- **`QUEUED`**: The remote program is in the Qiskit Function queue. The queue priority is based on how much you've used Qiskit Functions.\n",
     "- **`INITIALIZING`**: The remote program is starting; this includes setting up the remote environment and installing dependencies.\n",
-    "- **`RUNNING`**: The program is running.\n",
+    "- **`RUNNING`**: The program is running. This also includes several more detailed statuses if supported by specific functions\n",
+    "    - **`RUNNING: MAPPING`**\": The function is currently mapping your classical inputs to quantum inputs\n",
+    "    - **`RUNNING: OPTIMIZING_FOR_HARDWARE`**\": The function is optimizing for the selected QPU. This could include circuit transpilation, QPU characterization, observable backpropagation, etc.\n",
+    "    - **`RUNNING: WAITING_FOR_QPU`**: The function has submitted a job to Qiskit Runtime, and is waiting in the queue\n",
+    "    - **`RUNNING: EXECUTING_QPU`**: The function has an active Qiskit Runtime job\n",
+    "    - **`RUNNING: POST_PROCESSING`**: The function is post-processing results. This could include error mitigation, mapping quantum results to classical, etc.\n",
```

Comment:
```suggestion
    "    - **`RUNNING: OPTIMIZING_FOR_HARDWARE`**\": The function is optimizing for the selected QPU. This could include circuit transpilation, QPU characterization, observable backpropagation, and so forth\n",
    "    - **`RUNNING: WAITING_FOR_QPU`**: The function has submitted a job to Qiskit Runtime, and is waiting in the queue\n",
    "    - **`RUNNING: EXECUTING_QPU`**: The function has an active Qiskit Runtime job\n",
    "    - **`RUNNING: POST_PROCESSING`**: The function is post-processing results. This could include error mitigation, mapping quantum results to classical, and so forth\n",
```